src/games/strategy/triplea/ai/proAI/ProCombatMoveAI.java

```diff
@@ -967,12 +946,12 @@ private void determineUnitsToAttackWith(final Map<Territory, ProTerritory> attac
         final int isCanHold = canHold ? 1 : 0;
         final int isCantHoldAmphib = !canHold && !patd.getAmphibAttackMap().isEmpty() ? 1 : 0;
         final int isFactory = ProMatches.territoryHasInfraFactoryAndIsLand(player).match(t) ? 1 : 0;
-        final int isFFA = utils.isFFA(data, player) ? 1 : 0;
+        final int isFFA = ProUtils.isFFA(data, player) ? 1 : 0;
```

Comment:
Agree. A large portion of the methods are too long and should be refactored to shorter methods. This makes it easier to read and maintain and limits the need for adding comments. But the change your describe here is way out of scope as I just change the method reference to static.